### DISCUSSION TITLE
Using pgcrypto and pgp_sym_encrypt and pgp_sym_decrypt

### DISCUSSION THREAD

**Discussion by GeeWee** [prisma/prisma#7798] [Q&A]
Hey y'all.

I was wondering if there is any way to use Prisma and pgcrypto together. I want to encrypt some columns using pgcrypto, e.g. running a statement like the following:

```sql
INSERT INTO 'table' VALUES pgp_sym_encrypt('someEmail', 'myClientSidePassword')
SELECT pgp_sym_decrypt(email, 'myClientSidePassword') from 'table'
```

Is there any way to do this without having to dive down into `queryRaw`?

**Comment by ryands17** [comment]
@GeeWee 👋 

Unfortunately not, Prisma doesn't support external extensions like this yet directly so `queryRaw` is the only way. It would be great if you could open a [feature request](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=) with your use case so that we can look into this 🙂

**Comment by droriv-fijoya** [comment]
@ryands17 👋  
Any chance this added?
I saw some docs about supporting other functionalities from pgcrypto but didn't find docs about this specific functionality
tnx!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
